src/generic/dbgrptg.cpp

```diff
@@ -75,7 +75,7 @@ wxDumpPreviewDlg::wxDumpPreviewDlg(wxWindow *parent,
                                    const wxString& text)
                 : wxDialog(parent, wxID_ANY, title,
                            wxDefaultPosition, wxDefaultSize,
-                           wxDEFAULT_DIALOG_STYLE | wxRESIZE_BORDER)
+                           wxDEFAULT_DIALOG_STYLE | wxRESIZE_BORDER | wxMINIMIZE_BOX | wxMAXIMIZE_BOX)
```

Comment:
Does it really make sense to make this model dialog minimizable? IMHO this is a rather unusual thing to do, why do you think it would be useful?

Reply:
I agree they are not really useful, especially in `wxDebugReportDialog`. Maximizing is useful in `wxDumpPreviewDlg` so you don't have to manually resize the dialog to make the stack trace readable. But using `Open...` instead of `View...`, and use the default XML viewer, is a lot more useful.

Thanks for merging.